guides/common/assembly_configuring-a-freeipa-server-as-an-external-identity-provider-for-project.adoc

```diff
@@ -1,7 +1,15 @@
 include::modules/con_configuring-a-freeipa-server-as-an-external-identity-provider-for-project.adoc[]
 
-include::modules/con_using-freeipa.adoc[leveloffset=+1]
+include::modules/proc_enrolling-project-server-in-freeipa-domain.adoc[leveloffset=+1]
 
-include::modules/proc_configuring-freeipa-authentication-on-server.adoc[leveloffset=+1]
+include::modules/proc_configuring-the-freeipa-authentication-source-on-projectserver.adoc[leveloffset=+1]
 
-include::modules/proc_configuring-host-based-authentication-control.adoc[leveloffset=+1]
+include::modules/proc_configuring-host-based-access-control-for-freeipa-users-logging-in-to-foreman.adoc[leveloffset=+1]
```

Comment:
"...-to-project"? I am totally ok always saying foreman/smart-proxy in file names.